packages/itwinui-react/src/core/utils/hooks/useIsomorphicLayoutEffect.tsx

```diff
@@ -9,5 +9,5 @@ import * as React from 'react';
  *
  * @see https://fb.me/react-uselayouteffect-ssr
  */
-export const useIsomorphicLayoutEffect =
+export const useLayoutEffect =
```

Comment:
this might be a nit, but i think it would be more clear to keep the original name and rename it when exporting

```jsx
const useIsomorphicLayoutEffect =
  typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;

export { useIsomorphicLayoutEffect as useLayoutEffect };
```

Reply:
Additionally, you could also add a short comment to explain why we are exporting it with a different name.

Reply:
Changed. Comment was added to the larger comment about the function.

Reply:
Looks good.

Just a note about PR etiquette: please refrain from self-resolving PR comments. Let the reviewer verify the change and wait for them to resolve it.